src/Components/App.js

```diff
@@ -12,25 +12,35 @@ import ItemExpanded from "./ItemExpanded";
 import Login from "./Login";
 import NoMatch from "./NoMatch";
 
-import response from "../data/allItems.json";
+const GET_ITEMS = gql`
+  query getItems {
+   items: getItems {
+    id
+    name
+    description
+    category
+    status
+    available
+    amount
+    userId
+   }
+  }
+`;
 
 function App() {
-
-  const [itemData, setItemData] = useState([]);
-   
-  useEffect(() => {
-     setItemData(response.data.getItems);
-  }, []);
+  const { loading, error, data } = useQuery(GET_ITEMS); 
 
   const findItem = (id) => {  
-    return itemData.find(item => item.id === id)
+    return data.items.find(item => item.id === id)
   }
-
+  
+  if (loading) return <p>Loading...</p>;
+  if (error) return <p>Error</p>;
```

Comment:
Future note: We will come back and update the error handling here to be more robust on a later date.